flink-python/pyflink/datastream/state.py

```diff
@@ -925,14 +926,14 @@ class RocksdbCompactFilterCleanupStrategy(CleanupStrategy):
 
             def __init__(self,
                          query_time_after_num_entries: int,
-                         periodic_compaction_time=Time.days(30)):
+                         periodic_compaction_time=Duration.of_days(30)):
```

Comment:
`Duration.of_days(30)-> None` and do the similar thing to `cleanup_in_rocksdb_compact_filter`